crates/primitives/src/transaction/mod.rs

```diff
@@ -278,6 +278,43 @@ impl Transaction {
         }
     }
 
+    // TODO: dedup with effective_tip_per_gas
```

Comment:
yeh, I'd like to dedup this and unify, naming, because everything surrounding fee/price can be incredibly confusing

good with a followup for this